src/commands/deploy.ts

```diff
@@ -237,9 +241,32 @@ export default class Deploy extends DeployCommand {
     await execa('docker-compose', compose_args, { stdio: 'inherit' });
   }
 
+  private readValuesFile(values_file_path: string | undefined) {
+    let component_values = {};
+    if (values_file_path && fs.statSync(values_file_path)) {
+      const values_file_data = fs.readFileSync(values_file_path);
+      try {
+        component_values = JSON.parse(values_file_data.toString('utf-8'));
```

Comment:
remove support for JSON - create a generic loadYaml method?

Reply:
feels like we have a couple custom loadYaml methods throughout the code base for no reason

Reply:
Should that also check that the file exists? Or do we want to limit that function just to reading and returning contents?

Reply:
It shouldn't check if the file exists.
- If the user didn't pass a values param don't try to read
- if the user did pass the values param assume the file exists and fail if it doesn't